assets/js/components/ChangeBadge.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * ChangeBadge component.
+ *
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import { calculateChange, numFmt } from '../util';
+
+const ChangeBadge = ( { previousValue, currentValue } ) => {
```

Comment:
What is the point of using an anonymous functions for components? I know that we have many of them in our codebase and some engineers are constantly use this approach for all components, but what are benefits of this approach? 🤔 

PS: not asking to change this, just wondering why people like this approach.